pyfr/plugins/sampler.py

```diff
@@ -121,7 +214,7 @@ def _process_samples(self, samps):
             samps = self.elementscls.con_to_pri(samps.T, self.cfg)
             samps = np.array(samps).T
 
-        return samps.tolist()
+        return np.ascontiguousarray(samps, dtype=float)
```

Comment:
Just asking, is the use of float intentional here to reduce the file size?

Reply:
`dtype=float` refers to a Python float which is double (`np.float` is now deprecated).  Ensuring everything is a double is important as we're now using Gatherv which depends on the precise type of the array.